Observer/NewShipment.php

```diff
@@ -68,21 +69,20 @@ class NewShipment implements ObserverInterface
     public function __construct(MagentoOrderCollection $orderCollection = null)
     {
         $this->objectManager   = ObjectManager::getInstance();
-        $this->request         = $this->objectManager->get('Magento\Framework\App\RequestInterface');
-        $this->redirectFactory = $this->objectManager->get('Magento\Framework\Controller\Result\RedirectFactory');
-        $this->messageManager  = $this->objectManager->get('Magento\Framework\Message\Manager');
+        $this->request         = $this->objectManager->get(RequestInterface::class);
+        $this->redirectFactory = $this->objectManager->get(RedirectFactory::class);
+        $this->messageManager  = $this->objectManager->get(Manager::class);
         $this->orderCollection = $orderCollection ?? new MagentoOrderCollection($this->objectManager, $this->request);
-        $this->helper          = $this->objectManager->get('MyParcelNL\Magento\Helper\Data');
-        $this->modelTrack      = $this->objectManager->create('Magento\Sales\Model\Order\Shipment\Track');
```

Comment:
Moet dit weg?

Reply:
Het ‘moet’ niet weg, maar omdat die niet gebruikt werden, vond ik het zonde om die instantie (modelTrack) te maken. Die Data is er nog wel maar met use import dinges dus niet FQCN.